### ISSUE TITLE
ROBOT remove + --term-file removes whole ontology if termset contains no ontology terms

### ISSUE THREAD

**Issue by matentzn** [ontodev/robot#506]
When using remove in conjunction with --term-file, the whole ontology gets removed when no term in the term file is present in the input ontology. The correct behaviour should be that nothing is removed, right? Use case: Use sparql query to find bad classes. Remove bad classes. SPARQL query does not return bad classes or only one that is already stripped out by another robot command.

Unit test:

```
git clone https://github.com/monarch-ebi-dev/robot_test.git
cd robot_test
make fail_remove_termfile
```

(needs robot 1.4.0)

**Comment by jamesaoverton** [comment]
You're right, that's not the intended behaviour.

As the docs say, if no terms are specified, then `remove` assumes you want to start with all terms and then use `--select` to prune your target set. In this case you *are* specifying terms to remove, they just don't happen to be in the ontology. On this line we're erasing that distinction:

https://github.com/ontodev/robot/blob/master/robot-command/src/main/java/org/obolibrary/robot/RemoveCommand.java#L114

So only if `entityIRIs` is empty should we add all objects (as on L164).

**Comment by jamesaoverton** [comment]
Fixed by #507
